index.html

```diff
@@ -6,10 +6,176 @@
     <link rel="stylesheet" href="./style.css">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
-    <title>RG Conference-2022</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,400;1,600&display=swap" rel="stylesheet">
+    <title>CC Summit-2022</title>
 </head>
 <body>
-    <h1>RG Conference - 2022</h1>
+    <div class="header-page">
+        <div class="mobile-menu container-fluid pt-5 gap-5 bg-dark">
+            <ul class="menu gap-1 fs-5 p-5">
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#main-program">Program</a></li>
+                <li><a href="#main-program">Join</a></li>
+                <li><a href="#partner">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="./about.html">Contact</a></li>
+            </ul>
+        </div>
+        <div class="desk-header-menu justify-content-around">
+            <a href="./index.html"><img class="header-logo" src="./image/headerLogo.png" alt="logo"></a>
+            <div class="desktop-menu d-flex">
+                <div class="menu-desktop d-flex gap-5 p-4">
+                    <a href="./index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#main-program">Program</a>
+                    <a href="#main-program">Join</a>
+                    <a href="#partner">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="./about.html">Contact</a>
+                </div>
+            </div>
+        </div>
+        <div class="top container-fluid pt-4 p-4">
+            <header class="header position-fixed">
+                <div class="bar" id="bar1"></div>
+                <div class="bar" id="bar2"></div>
+                <div class="bar" id="bar3"></div>
+            </header>
+            <main class="main">
+                <div class="main-content pt-5">
+                    <h2>"Hello Sharing World"</h2>
+                    <H1>CARBON CAPTURE SUMMIT - 2022</H1>
+                </div>
+                
+                <p class="border border-3 border-white">Welcome to the Carbon Capture Summit 2022, where industry leaders will meet in Amsterdam to explore next-generation technologies for carbon capture, utilisation, storage, and transportation to meet net-zero emissions targets.</p>
+                <h2 class="text-dark">2022.11.24(THR) ~ 25(FRI)</h2>
+                
+                <h5>National Museum, Van Der Valk, Amsterdam, Netherlands</h5>
+            </main>
```

Comment:
- [ ] Kindly use section tags since they hold more semantic meaning in the given context. Instead of having a `main` and then `div`'s to structure your website, you should make sure the Headline, Speakers and Program sections are all enclosed in a `section` tag instead so we can infer they are different parts of the same project.

- [ ] Please increase the height of your Headline section so it conforms to the spacing guidelines of the Behance design
> Yours
![image](https://user-images.githubusercontent.com/90556221/183042005-fe97c609-0d3f-48d3-929d-e2a73bc14c6a.png)

> Behance
![image](https://user-images.githubusercontent.com/90556221/183042710-abbdb15f-482c-4089-983b-a536105016fb.png)

- [ ] Additionally, also ensure the date which I have highlighted above is bold in your design and has the same styling as the Behance design

- [ ] Lastly, make sure the paragraph describing the event has a fixed width. Right now it takes up as much space as it can unlike the Behance design which makes it as long as the heading.

Reply:
Added section tag.

Reply:
Adjusted the height and the width of the paragraph